fluent-package/yum/systemd-test/install-newly.sh

```diff
@@ -46,6 +46,9 @@ test -e /var/log/fluent/fluentd.log
 sudo $DNF remove -y fluent-package
 sudo systemctl daemon-reload
 
+getent passwd fluentd >/dev/null
+getent group fluentd >/dev/null
+
```

Comment:
I'm sorry.
I was suggesting the wrong file.
It should be `yum/systemd-test/install-newly.sh`, not `yun/install-test.sh`.

```suggestion
case $1 in
  local)
    getent passwd fluentd >/dev/null
    getent group fluentd >/dev/null
    ;;
  *)
    # TODO: Remove this branch after the following fix is applied to the latest release.
    # https://github.com/fluent/fluent-package-builder/pull/598
    (! getent passwd fluentd >/dev/null)
    (! getent group fluentd >/dev/null)
    ;;
esac
```

Reply:
Ah, got it. Thanks!